cmd/kind/get/kubeconfig/kubeconfig.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package kubeconfig implements the `kubeconfig` command
+package kubeconfig
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kind/pkg/cluster"
+	clusternodes "sigs.k8s.io/kind/pkg/cluster/nodes"
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+type flagpole struct {
+	Name string
+}
+
+// NewCommand returns a new cobra.Command for getting the kubeconfig with the internal node IP address
+func NewCommand() *cobra.Command {
+	flags := &flagpole{}
+	cmd := &cobra.Command{
+		Args:  cobra.NoArgs,
+		Use:   "kubeconfig",
+		Short: "prints cluster kubeconfig",
+		Long:  "prints cluster kubeconfig with internal IP address",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runE(flags, cmd, args)
+		},
+	}
+	cmd.Flags().StringVar(
+		&flags.Name,
+		"name",
+		cluster.DefaultName,
+		"the cluster context name",
+	)
+	return cmd
+}
+
+func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
+	// List nodes by cluster context name
+	n, err := clusternodes.ListByCluster("label=io.k8s.sigs.kind.role=control-plane")
+	if err != nil {
+		return err
+	}
+	nodes, known := n[flags.Name]
+	if !known {
+		return errors.Errorf("unknown cluster %q", flags.Name)
+	}
+	// grab kubeconfig version from one of the control plane nodes
+	cmdNode := nodes[0].Command("cat", "/etc/kubernetes/admin.conf")
```

Comment:
The default output should not be raw, it should be what we'd write to disk normally when creating the cluster

Reply:
Can you elaborate more?
I did it so you can pipe the output 
```
$ ./kind get kubeconfig  >  test.txt
$ cat test.txt
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwK
Y201bGRHVnpNQjRYRFRFNU1EVXdNekV3TVRZME5Wb1hEVEk1TURRek1ERXdNVFkwTlZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBREND
QVFvQ2dnRUJBTjBlClJ3RElkTEF2Vll4SEpKeDhyeFFWME5VcjZ4OHlTczgwN1RGbWhBWDRqbDFJSFZ6Q0pFM3pNd2E2WDM0WXpaTTcKbUVPeC9ySEpTelFtMEZVRWhZSlN5UzIwQWZWbTdkOGZrV3yVmxPZkMK
U3VXbHl3cExUSU9DRTQ1NVpwNGFzek1TeUEvakdiUi8rSklpUmZpTFNWa0RXdTU1VlpGUUdLNEp4V05zRjJhcwo0WSt2dGVRUWlKdGZ6K3N5d09hZU5lTFhyZWNKaU9LRkRZeVJqenJibjhKaDEvRUE2Ym1P
WkZLdUk5THlUcTgyCnpmLy9SYnlYVmxwZGpzRXhQNFN5dFlSWE1KSTJsS2VybjBpTnozOVp5ZlhRL3ozT3kyZERqUm9JK2lkMEVqak8KcEFkblhKZ3lZK0tnSjYxZ3hERGFZcDZmZk9ZOFUzbnhsVjUydGpS
    server: https://172.17.0.2:6443
  name: kind
contexts:
- context:
    cluster: kind
    user: kubernetes-admin
  name: kubernetes-admin@kind
current-context: kubernetes-admin@kind
kind: Config
preferences: {}
users:
- name: kubernetes-admin
  user:
```

Reply:
we should have two modes:
- the same contents that kind would write during `kind create cluster`
- the raw contents from within the cluster, no kind specific modifications
   - I would rename "raw" to "internal", to clarify the purpose, we may still tweak it for other reasons in the future (perhaps rewriting the context name, user...)

Reply:
done